src/utils.ts

```diff
@@ -10,6 +11,10 @@ export function isAlpha(char: string): boolean {
   return ALPHA.test(char);
 }
 
+export function isUnicode(char: string): boolean {
```

Comment:
Small nitpick: let's call this something like `isExtendedUnicode` because every string is unicode.